lib/mix/tasks/docs.ex

```diff
@@ -349,6 +352,16 @@ defmodule Mix.Tasks.Docs do
     for formatter <- get_formatters(options) do
       index = generator.(project, version, Keyword.put(options, :formatter, formatter))
       Mix.shell().info([:green, "View #{inspect(formatter)} docs at #{inspect(index)}"])
+
+      if List.keymember?(options, :warnings_as_errors, 0) and ExDoc.WarningCounter.count() > 0 do
```

Comment:
I think it is clearer in intention to use `Keyword.has_key?/2` since options is a keyword.

```suggestion
      if Keyword.has_key?(options, :warnings_as_errors) and ExDoc.WarningCounter.count() > 0 do
```